pom.xml

```diff
@@ -40,7 +40,7 @@
   <scm>
     <connection>scm:git:https://${env.GITHUB_TOKEN_USR}:${env.GITHUB_TOKEN_PSW}@github.com/camunda/camunda.git</connection>
     <developerConnection>scm:git:https://${env.GITHUB_TOKEN_USR}:${env.GITHUB_TOKEN_PSW}@github.com/camunda/camunda.git</developerConnection>
-    <tag>8.6.13</tag>
+    <tag>8.6.0</tag>
```

Comment:
>and sets repo tags to the. 8.6.0 like done in usual generated commits too, like [here for 8.5](https://github.com/camunda/camunda/pull/30494/commits/5e15c9c810b002d09f9cf3c6cefc375d85bcc802). I lack to understand why but just replicated maven release behavior here.

😅 This is so strange.

🔧 On [`stable/8.7`](https://github.com/camunda/camunda/blob/stable/8.7/pom.xml#L43) we've tagged `8.7.0-rc3`. We might want to align this (or investigate what the tag should actually be during development).

https://github.com/camunda/camunda/blob/6f81e27454bb58ad7c30665eaf13ac4a8a8dbe1a/pom.xml#L43